src/components/elements/query-image.tsx

```diff
@@ -22,32 +21,19 @@ export type ImageWrapperProps = {
     width: string
     height: string
     className?: string
-    loading: 'eager' | 'lazy'
```

Comment:
Can you remove `width`, `height`, `className` and other properties that are already declared by the `IGatsbyImageData`? Change this to an interface and extend 

```
export interface QueryImageProps extends IGatsbyImageData
```